### DISCUSSION TITLE
docsify+giscus.md

### DISCUSSION THREAD

**Discussion by giscus** [xdd1997/docsify#25] [General]
# docsify+giscus.md

https://www.20161101.xyz/docsify/#/md_File/docsify+giscus

**Comment by xdd1997** [comment]
注意：`data-category`与`data-category-id`要对应，不能随便写，评论区的显示与`crossorigin`无关
`General`对应`DIC_kwDOIg7OMc4CSz3W`
`Announcements`对应`DIC_kwDOIg7OMc4CSz3V`

**Reply by xdd2026** [reply]
ok

**Comment by xdd2026** [comment]
2233
